apps/web/src/app/recovery/reset-password/request/page.tsx

```diff
@@ -65,8 +63,8 @@ export default function RequestResetPage() {
         <Button
           type="submit"
           className="w-full"
-          disabled={isLoading}>
-          {isLoading ? 'Sending...' : 'Send Recovery Email'}
+          disabled={isPending}>
+          {isPending ? 'Sending...' : 'Send Recovery Email'}
```

Comment:
there is also a loading property on the button, use that instead of disable while using loading state